src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -97,9 +100,48 @@ public IEnumerable<IEvaluation> AnalyzeTemplate(string template, string paramete
                     throw new TemplateAnalyzerException(BicepCompileErrorMessage, e);
                 }
             }
+            IEnumerable<IEvaluation>  evaluations = DeepAnalyzeTemplate(template, parameters, templateFilePath, template, 0, isBicep, sourceMap);
```

Comment:
Can you rename the function for something a bit more declarative? "AnalyzeAllIncludedTemplates" or something like that